docs/src/api.md

```diff
@@ -11,10 +11,10 @@ SSAStepper
 
 ## Types of Jumps
 ```@docs
-MassActionJump
 ConstantRateJump
+MassActionJump
 VariableRateJump
-RegularJump
```

Comment:
Just doesn't have a doc string at the moment. Being added in https://github.com/SciML/DiffEqJump.jl/pull/248/files so we can add it back shortly... (I didn't want to add one since it would come in through there...)